react/src/pages/utils.tsx

```diff
@@ -22,6 +22,7 @@ const months = [
 /*****   TYPINGS   *****/
 export type Counts = { [key: string]: number };
 export type KeyValue = { [key: string]: string };
+export type Variant = "button" | "caption" | "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "inherit" | "subtitle1" | "subtitle2" | "body1" | "body2" | "overline" | "srOnly";
```

Comment:
I believe you can just use `Typography["variant"]` to get the type instead of duplicating it here.